backend/dataall/cdkproxy/stacks/__init__.py

```diff
@@ -2,7 +2,6 @@
 from .environment import EnvironmentSetup
 from .pipeline import PipelineStack
 from .manager import stack, instanciate_stack, StackManager
-from .notebook import SagemakerNotebook
```

Comment:
[Clarification ] No need to load it, we remove it because we load it with loader